internal/hosting/hostingdomain/unsupported_service_error.go

```diff
@@ -8,6 +8,7 @@ func UnsupportedServiceError() error {
 
 This command requires hosting on one of these services:
 * Bitbucket
+* Bitbucket datacenter
```

Comment:
Thanks for updating this error message. Please also update the end-to-end test that documents/verifies it. It's in features/repo/edge_cases/unsupported_hosting_service.feature.

Reply:
Hm strange, I had another one failing but not this one.

`features/propose/edge_cases/unsupported_hosting_service.feature`

I updated the test you indicated and ran `make cuke` but nothing changed.